src/main/java/io/github/moulberry/notenoughupdates/overlays/EquipmentOverlay.java

```diff
@@ -378,6 +383,70 @@ public void renderPets(GuiInventory inventory, int mouseX, int mouseY, int width
 		}
 	}
 
+	private final Map<ItemStack, Integer> itemsToAdd = new HashMap<>();
+
+	@SubscribeEvent
+	public void onClickItem(PlayerInteractEvent event) {
+		if (event.action == PlayerInteractEvent.Action.LEFT_CLICK_BLOCK || Minecraft.getMinecraft().thePlayer.getHeldItem() == null) return;
+
+		ItemStack heldItem = Minecraft.getMinecraft().thePlayer.getHeldItem();
+		NBTTagList heldItemLore = heldItem.getTagCompound().getCompoundTag("display").getTagList("Lore", 8);
```

Comment:
We have `ItemUtils.getLore(heldItem)` to decode item lore. I guess this is "faster" since we do only access one nbt tag instead of mapping all of them into a new line, but is that speed benefit worth it? In either case, we could also add support for something along the lines of a "ItemUtils.getItemType" utility for this.